astropy/nddata/tests/test_utils.py

```diff
@@ -148,6 +148,50 @@ def test_slices_nonfinite_position(position):
         overlap_slices((7, 7), (3, 3), position)
 
 
+def test_slices_limit_rounding_method():
```

Comment:
this looks like 5 different tests in a trench coat. Can you refactor it with `@pytest.mark.parametrize` ?